examples/disks-ephemeral/README.md

```diff
@@ -0,0 +1,7 @@
+# AKS Engine - Ephemeral Disks
+
+## Overview
+
+AKS Engine enables you to create customized Kubernetes cluster on Microsoft Azure with [ephemeral disks](https://docs.microsoft.com/en-us/azure/virtual-machines/linux/ephemeral-os-disks).
+
+These examples are provided as a reference, note that managed disks is the default storage account type if none is specified.

```

Comment:
Should we add any copy that describes operational scenarios that would recommend an ephemeral disk configuration? Or do we need to do more discovery before publishing recommendations like that?

Reply:
I'm not ready to talk about specific workloads, but I did create a section on the features page for this that describes the state and risks.